src/view/screens/LanguageSettings.tsx

```diff
@@ -302,7 +309,51 @@ export function LanguageSettingsScreen(_props: Props) {
             </Text>
           </Button>
         </View>
-      </View>
+
+        {/* NATIVE TRANSLATION */}
+        {isNativeTranslationAvailable && (
+          <View style={{paddingBottom: 20}}>
+            <Divider style={{marginBottom: 20}} />
+
+            <Text type="title-sm" style={[pal.text, s.pb5]}>
+              <Trans>Native Translations</Trans>
+            </Text>
+            <Text style={[pal.text, s.pb10]}>
+              <Trans>
+                Translations are provided the operating system, where available.
```

Comment:
```suggestion
                Translations are provided by the operating system, where available.
```
I think **by** is missing here